nuxeo-platform-video-convert/pom.xml

```diff
@@ -60,7 +61,8 @@
         <configuration>
           <forkMode>pertest</forkMode> <!-- this is what triggers the problem, having it to "never" works also -->
           <systemPropertyVariables>
-              <log4j.configuration>${file.uri.prefix}${project.build.directory}/test-classes/log4j.properties</log4j.configuration>
+            <log4j.configuration>${file.uri.prefix}${project.build.directory}/test-classes/log4j.properties
+            </log4j.configuration>
```

Comment:
Unneeded formatting change when the whole file doesn't contain any change. Undo all changes on this `pom.xml`.